tools/dotnet-linker/SetupStep.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+using Mono.Linker.Steps;
+
+namespace Xamarin {
+
+	public class SetupStep : BaseStep {
+
+		protected override void Process ()
+		{
+			Console.WriteLine ("Hello SetupStep");
```

Comment:
maybe add a comment this is temporary ?
or a README.md in the directory ?
normally `master` is releasable - and that's not the case for this code yet